doc/source/whatsnew/v1.2.0.rst

```diff
@@ -322,7 +322,7 @@ Reshaping
 - Bug in :meth:`DataFrame.pivot_table` with ``aggfunc='count'`` or ``aggfunc='sum'`` returning ``NaN`` for missing categories when pivoted on a ``Categorical``. Now returning ``0`` (:issue:`31422`)
 - Bug in :func:`union_indexes` where input index names are not preserved in some cases. Affects :func:`concat` and :class:`DataFrame` constructor (:issue:`13475`)
 - Bug in func :meth:`crosstab` when using multiple columns with ``margins=True`` and ``normalize=True`` (:issue:`35144`)
--
+- Bug in :meth:`DataFrame.stack` for empty DataFrame (:issue:`36113`)
```

Comment:
can you elaborate a bit on what is changing.

Reply:
I have elaborate this a bit in the latest commit